OpenRA.Mods.Common/Traits/Modifiers/FrozenUnderFog.cs

```diff
@@ -120,6 +123,13 @@ public void Tick(Actor self)
 					frozenActor.TooltipInfo = tooltip.Value.TooltipInfo;
 					frozenActor.TooltipOwner = tooltip.Value.Owner;
 				}
+
+				if (ownerHasChanged[player])
+				{
+					var mod = self.TraitsImplementing<IRadarColorModifier>().FirstOrDefault();
+					frozenActor.RadarColor = mod != null ? mod.RadarColorOverride(self) : self.Owner.Color.RGB;
+					ownerHasChanged[player] = false;
+				}
```

Comment:
We don't want to change the color until the player for who that actor is frozen reveals it again and sees that it was captured.
Otherwise frozen actors will change colors under the fog, giving away information that you shouldn't know. (The commit message describes that, maybe I should have put a comment in there as well).